lib/internal/bootstrap/switches/does_not_own_process_state.js

```diff
@@ -16,3 +23,16 @@ if (credentials.implementsPosixCredentials) {
 
 // ---- keep the attachment of the wrappers above so that it's easier to ----
 // ----              compare the setups side-by-side                    -----
+
+const {
+  codes: { ERR_WORKER_UNSUPPORTED_OPERATION }
+} = require('internal/errors');
+
+function wrappedUmask(mask) {
+  // process.umask() is a read-only operation in workers.
+  if (mask !== undefined) {
+    throw new ERR_WORKER_UNSUPPORTED_OPERATION('Setting process.umask()');
```

Comment:
Note: this reads `Setting process.umask() is not supported in workers`, apart from that we need to get rid of the worker part from the message, maybe `calling process.umask() as a setter` would be less ambiguous (I thought it was about `process.umask = something` from the first glance)

Reply:
Should this be changed before landing?

Reply:
I don’t think so – it’s unrelated to the changes here.